tests/k8-cluster/main.yaml

```diff
@@ -2,47 +2,53 @@
 #
 #  This playbook creates a kubernetes cluster on a single system using the
 #  example in the Red Hat documentation.  The cluster contains a database
-#  and a webserver pod with data from the database to present a webpage that 
+#  and a webserver pod with data from the database to present a webpage that
 #  says "Red Hat Rocks!"
-# 
+#
 #  Execution Steps:
 #    1. Subscribe to RedHat if running RHEL Atomic Host
-#    2. Create an insecure private registry 
+#    2. Create an insecure private registry
 #    3. Copy over the mariadb and apache webserver Dockerfile and supporting
 #       files, build the docker image, tag the image, then push to the private
 #       registry.
 #    4. Setup and start the kubernetes pods
 #    5. Setup and start the mariadb and webserver pods
-#    6. Verify that the pods are running and that the mariadb and webserver 
+#    6. Verify that the pods are running and that the mariadb and webserver
 #       can communicate through the kubernetes services
 #
 - name: k8-cluster
   hosts: all
   become: yes
 
   roles:
+    # This playbook requires Ansible 2.2 and an Atomic Host
+    - role: ansible_version_check

```

Comment:
I'm seeing `^M` characters delimiting the lines in this file.  Can you remove them and re-submit?

http://its.ucsc.edu/unix-timeshare/tutorials/clean-ctrl-m.html